lib/rules/declaration-property-value-no-unknown/index.js

```diff
@@ -84,10 +124,22 @@ const rule = (primary, secondaryOptions) => {
 
 			if (!isStandardSyntaxValue(value)) return;
 
-			if (isCustomProperty(prop)) return;
+			if (isCustomProperty(prop) && !typedCustomPropertyNames.has(prop)) return;
 
 			if (isPropIgnored(prop, value)) return;
 
+			// https://github.com/mdn/data/pull/674
+			// `initial-value` has a incorrect syntax definition.
+			// In reality everything is valid.
+			if (
+				/^initial-value$/i.test(prop) &&
+				decl.parent &&
+				isAtRule(decl.parent) &&
+				/^property$/i.test(decl.parent.name)
+			) {
+				return;
+			}
```

Comment:
I didn't cache the regexp's here because this entire block should be removed.

I don't mind optimizing the code anyway.
But I am really hoping that csstree will receive some attention in the near future.